### PR TITLE
Removed -SNAPSHOT

### PR DESCRIPTION
**Pull request by Nishith**

### CODE REVIEW

**Comment by sreevatsanraman** [comment]
There are more places please use 

`find . -type f  -name '*' | xargs grep SNAPSHOT | grep -v '.git' | grep -v '.iml'
`

File: cdap-ui/server/config/cdap-ui-config.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "tracker": {
     "artifact": {
       "name": "tracker",
-      "version": "0.4.0-SNAPSHOT",
+      "version": "0.4.0",
       "scope": "SYSTEM"
     },
     "appId": "_Tracker",
```
Comment (anwar6953) [inline cdap-ui/server/config/cdap-ui-config.json:8]: navigator, right below this, should be 0.5.0

Comment (Nishith) [reply cdap-ui/server/config/cdap-ui-config.json:8]: Done

**Comment by sreevatsanraman** [comment]
LGTM

**Pull request merged**